src/autopas/containers/linkedCells/traversals/LCC18Traversal.h

```diff
@@ -84,7 +87,8 @@ class LCC18Traversal : public C18BasedTraversal<ParticleCell, PairwiseFunctor, d
   /**
    * CellFunctor to be used for the traversal defining the interaction between two cells.
    */
-  internal::CellFunctor<typename ParticleCell::ParticleType, ParticleCell, PairwiseFunctor, dataLayout, useNewton3>
+  internal::CellFunctor<typename ParticleCell::ParticleType, ParticleCell, PairwiseFunctor, dataLayout, useNewton3,
+                        true>
```

Comment:
```suggestion
  internal::CellFunctor<typename ParticleCell::ParticleType, ParticleCell, PairwiseFunctor, dataLayout, useNewton3,
                        /*bidirectional*/ true>
```

Reply:
done in [8081014](https://github.com/AutoPas/AutoPas/pull/619/commits/80810142d34cf7872346976089a90e1753180bb7)